### PR TITLE
refactor(apps/code-examples): update code examples to be strict-mode compliant

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/879/storybook/)

Component Storybooks:

- _(no component storybooks affected in this pr)_ 

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/879/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/879/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/879/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/879/playground/)

File: apps/code-examples/src/app/code-examples/tiles/tiles/inline-help/tile-demo-tile2.component.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export class TileDemoTile2Component {
     alert('Help inline button clicked!');
   }
 
-  onHelpClick($event: MouseEvent) {
+  onHelpClick($event: MouseEvent): void {
```
Comment (Blackbaud-SteveBrush) [inline apps/code-examples/src/app/code-examples/tiles/tiles/inline-help/tile-demo-tile2.component.ts]:
```
public onHelpClick
```

File: apps/code-examples/src/app/code-examples/split-view/split-view/page-bound/split-view-page-bound-demo.component.ts [hunk 2]
```diff
@@ -147,7 +153,7 @@ export class SplitViewPageBoundDemoComponent {
     console.log('Denied clicked!');
   }
 
-  private loadFormGroup(record: any): void {
+  private loadFormGroup(record: SplitViewDemoRecord): void {
```
Comment (Blackbaud-SteveBrush) [inline apps/code-examples/src/app/code-examples/split-view/split-view/page-bound/split-view-page-bound-demo.component.ts]: `#loadFormGroup`

File: apps/code-examples/src/app/code-examples/split-view/split-view/basic/split-view-demo.component.ts [hunk 3]
```diff
@@ -99,7 +104,7 @@ export class SplitViewDemoComponent {
     console.log('Denied clicked!');
   }
 
-  private loadFormGroup(record: any): void {
+  private loadFormGroup(record: SplitViewDemoRecord): void {
```
Comment (Blackbaud-SteveBrush) [inline apps/code-examples/src/app/code-examples/split-view/split-view/basic/split-view-demo.component.ts]:
```
#loadFormGroup
```

File: apps/code-examples/src/app/code-examples/colorpicker/colorpicker/basic/colorpicker-demo.component.ts [hunk 4]
```diff
@@ -22,25 +22,19 @@ export class ColorpickerDemoComponent implements OnInit {
...
+    });
   }
 
   public onSelectedColorChanged(args: SkyColorpickerOutput): void {
     console.log('Reactive form color changed:', args);
   }
 
   public submit(): void {
-    const controlValue = this.reactiveForm.get('favoriteColor').value;
+    const controlValue = this.reactiveForm?.get('favoriteColor')?.value;
```
Comment (Blackbaud-SteveBrush) [inline apps/code-examples/src/app/code-examples/colorpicker/colorpicker/basic/colorpicker-demo.component.ts]: You can remove this first question mark.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Review by Blackbaud-CoreyArcher — APPROVED** [review]

**Pull request merged**